src/psbt.h

```diff
@@ -86,9 +86,9 @@ struct PSBTInput
             }
 
             // Write the sighash type
-            if (sighash_type > 0) {
+            if (sighash_type != std::nullopt) {
```

Comment:
In commit "psbt: Make signhash_type std::optional<int>"

Nit: I think the ideomatic way of writing this would be `if (sighash_type.has_value())` or just `if (sighash_type)` (here and further).